cmd/wallet-web/src/components/DIDManagement/ImportDid.vue

```diff
@@ -0,0 +1,158 @@
+<!--
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <div class="flex flex-col justify-start items-start py-8">
+    <div class="flex flex-col flex-grow justify-start items-start mb-5 w-full">
+      <label class="">Enter Digital Identity:</label>
```

Comment:
empty class here as well

Reply:
You commenting on outdated code @talwinder50